infra/feast-operator/internal/controller/services/util.go

```diff
@@ -94,6 +94,9 @@ func ApplyDefaultsToStatus(cr *feastdevv1alpha1.FeatureStore) {
 				pvc := services.Registry.Local.Persistence.FilePersistence.PvcConfig
 				if pvc.Create != nil {
```

Comment:
i wonder if this whole `pvc.Create` section would work as a separate function that all 3 service types could call?

Reply:
I added a function to consolidate this bit.